migrations/2020_03_19_migrate_cronjob_history_from_processengine_databases.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-require-imports */
+
+const fs = require('fs');
+const path = require('path');
+
+const SequelizeConnectionManager = require('@atlas-engine/sequelize_connection_manager').SequelizeConnectionManager;
+const BpmnModelParser = require('@atlas-engine/process_model_parser').BpmnModelParser;
+
+const ProcessEngineCronjobHistoryModel = require('@process-engine/persistence_api.repositories.sequelize').CronjobHistoryEntryModel;
+const ProcessEngineProcessDefintionModel = require('@process-engine/persistence_api.repositories.sequelize').ProcessDefinitionModel;
+
+const AtlasEngineCronjobHistoryModel = require('@atlas-engine/persistence.repositories.sequelize').CronjobHistoryEntryModel;
+
+const sequelizeConnectionManager = new SequelizeConnectionManager();
+
+const parser = new BpmnModelParser();
+
+module.exports = {
+  up: async (queryInterface, Sequelize, dialect, atlasEngineSequelizeOptions, processEngineSqliteStoragePath) => {
+
+    const processEngineDatabasesDoesNotExit = processEngineSqliteStoragePath === undefined;
+    if (processEngineDatabasesDoesNotExit) {
+      return;
+    }
+
+    const databasesExist = fs.existsSync(path.join(processEngineSqliteStoragePath, 'processengine.sqlite'))
+                        && fs.existsSync(path.join(processEngineSqliteStoragePath, 'process_model.sqlite'));
+
+    if (!databasesExist) {
+      return;
+    }
+
+    console.log('Migrating Cronjob History data from ProcessEngine');
+
+    const processEngineProcessEngineSequelizeInstanceConfig = getProcessEngineConfig(processEngineSqliteStoragePath, 'processengine');
```

Comment:
Ich weis, dass die DB Datei so heisst, ich würde hier trotzdem den fachlich korrekten Namen benutzen, damit Betrachter hier verstehen was gemeint ist.

Also z.B.: 

```suggestion
    const processEngineCronjobHistorySequelizeInstanceConfig = getProcessEngineConfig(processEngineSqliteStoragePath, 'processengine');
```